lib/msal-core/src/ClientInfo.ts

```diff
@@ -30,6 +30,12 @@ export class ClientInfo {
         this._utid = utid;
     }
 
+    static createClientInfoFromIdToken(oid:string): string {
```

Comment:
I think the method name may cause a little confusion given that the argument it receives is not really an idToken but a specific claim. Maybe it'd be better to receive the idToken and extract the subject/oid inside the method?

Reply:
I dont disagree. Thanks for the suggestion